public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -60,6 +72,10 @@ const ServiceAccountsListPage = ({
     updateServiceAccount(updatedServiceAccount);
   };
 
+  const onRemoveServiceAccount = (serviceAccount: ServiceAccountDTO) => {
```

Comment:
Could we remove this function and just call `removeServiceAccount` wherever it's needed?